.github/workflows/test.yml

```diff
@@ -12,13 +12,9 @@ jobs:
     strategy:
       matrix:
         node-version:
+          - 21
```

Comment:
Odd versions don't get LTS I think and become unsupported after 6 months. So I wouldn't include 21 here

Reply:
Fixed (and rebased)